docs/concepts/crds/repository/index.md

```diff
@@ -5,170 +5,105 @@ menu:
-
-```bash
-$ kubectl delete repository <repository-name>
-
-# Example
-$ kubectl delete repository gcs-demo-repo
-repository "gcs-demo-repo" deleted
-```
-
-This will delete only `Repository` crd. It won't delete any backed up data from the backend. You can recreate the `Repository` object later to reuse existing data as long as your restic password in unchanged.
-
->If you delete `Repository` crd while respective stash sidecar still exists on the workload, it will fail to take further backup.
-
-**Delete `Repository` along with backed up data :**
-
-In order to prevent the users from accidentally deleting backed up data, Stash uses a special `wipeOut` flag in `spec` section of `Repository` crd. By default, this flag is set to `wipeOut: false`. If you want to delete respective backed up data from backend while deleting `Repository` crd, you must set this flag to `wipeOut: true`.
-
-> Currently, Stash does not support wiping out backed up data for local backend. If you want to cleanup backed up data from local backend, you must do it manually.
-
-Here, is an example of deleting backed up data from GCS backend,
-
-- First, set `wipeOut: true` by patching `Repository` crd.
-
-  ```bash
-  $ kubectl patch repository gcs-demo-repo --type="merge" --patch='{"spec": {"wipeOut": true}}'
-  repository "gcs-demo-repo" patched
-  ```
-
-- Finally, delete `Repository` object. It will delete backed up data from the backend.
-
-  ```bash
-  $ kubectl delete repository gcs-demo-repo
-  repository "gcs-demo-repo" deleted
-  ```
-
-You can browse your backend storage bucket to verify that the backed up data has been wiped out.
+## Repository `Spec`
+`Repository` CRD has the following fields in the `.spec` section:
+- **spec.appRef :** `spec.AppRef` refers to the application that is being backed up in this `Repository`.
+- **spec.storageRef :** `spec.storageRef` refers to the `BackupStorage` CR which contain the storage information where the backed up data will be stored. The `BackupStorage` could be in a different namespace. However, the `Repository` namespace must be allowed to use the `BackupStorage`.
+- **spec.path :** `spec.path` represents the directory inside the `BackupStorage` where this Repository is storing its data. This path is relative to the path of `BackupStorage`. This path must be **unique** for each `Repository` referring same `BackupStorage`.
+- **spec.deletionPolicy :** `spec.deletionPolicy` specifies what to do when a `Repository` CR is deleted. The valid values for this field are:
+  - **Delete :** This will delete the respective `Snapshot` CRs from the cluster but keep the backed up data in the storage. This is the default behavior.
+  - **WipeOut :** This will delete the respective `Snapshot` CRs as well as the backed up data from the storage.
+- **spec.encryptionSecret :** refers to the `Secret` containing the encryption key which will be used to encode/decode the backed up data. You can refer to a `Secret` of a different namespace.
```

Comment:
```suggestion
  - **Delete :** This will delete the respective Repository CR and associated `Snapshot` CRs from the cluster but keep the backed up data in the storage. This is the default behavior.
  - **WipeOut :** This will delete the respective Repository CR, associated `Snapshot` CRs and the backed up data in `spec.path` from the storage.
- **spec.encryptionSecret :** refers to the `Secret` containing the encryption key which will be used to encode/decode the backed up data. You can refer to a `Secret` of a different namespace.
```